packages/backend/src/plugins/catalog.ts

```diff
@@ -26,6 +27,19 @@ export default async function createPlugin(
     }),
   );
 
+  builder.addEntityProvider(
+    GithubEntityProvider.fromConfig(env.config, {
+      logger: env.logger,
+      // optional: alternatively, use scheduler with schedule defined in app-config.yaml
+      schedule: env.scheduler.createScheduledTaskRunner({
+        frequency: { minutes: 30 },
+        timeout: { minutes: 3 },
+      }),
+      // optional: alternatively, use schedule
+      //scheduler: env.scheduler,
```

Comment:
Any reason to disable this?

Reply:
it's a either or thing with the filed above. and this one was erring out